stripe-crypto-onramp/src/main/java/com/stripe/android/crypto/onramp/di/OnrampComponent.kt

```diff
@@ -36,6 +37,9 @@ internal interface OnrampComponent {
         @BindsInstance
         fun lifecycleOwner(lifecycleOwner: LifecycleOwner): Builder
 
+        @BindsInstance
+        fun componentActivity(activity: ComponentActivity): Builder
```

Comment:
(not for this PR) We should prefer using [`Component.Factory`](https://dagger.dev/api/2.28/index.html?dagger/Component.Factory.html) instead of `Component.Builder` since it has better compile-time guarantees.